tests/integration_tests/flows/test_kafka.py

```diff
@@ -30,32 +28,38 @@
 STREAM_OUT_TS = f"test_stream_out_ts_{STREAM_SUFFIX}"
 DS_NAME = "kafka_test_ds"
 
-def read_stream(stream_name, buf, stop_event):
```

Comment:
With default Kafka configuration (when no groups exist). A consumer must be launched before a producer.
Otherwise all data written before consumer starts would be lost.